test/e2e/util.go

```diff
@@ -61,6 +61,22 @@ func GetPod(namespace, namePrefix, containsImage string, kubeclient kubernetes.I
 	return nil, nil
 }
 
+// GetPodLogs retrieves logs of a given pod
+func GetPodLogs(pod *v1.Pod, kubeclient kubernetes.Interface) (string, error) {
+	req := kubeclient.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &v1.PodLogOptions{Timestamps: true})
+	logs, err := req.Stream()
+	if err != nil {
+		return "", xerrors.Errorf("could not get logs for pod %q: %w", pod.Name, err)
+	}
+	buf := &bytes.Buffer{}
+	_, err = io.Copy(buf, logs)
```

Comment:
Can we use [io/ioutil.ReadAll](https://golang.org/pkg/io/ioutil/#ReadAll) here?

(I realize I wrote this code originally... I also need my code reviewed :/ )